grafana/README.md

```diff
@@ -41,3 +41,67 @@ jb install
 ```sh
 ./scripts/deploy.sh
 ```
+
+---
+
+# Adding a new Codified dashboard
+
+1. Create a new `jsonnet` file in `grafana/src`.
+2. Import the required depencencies for your dashboard. You can get an idea of what this looks like from the other `jsonnet` dashboards.
+3. Create panels.
+
+---
+
+## Adding a new panel:
+
+Read the [Grafonnet Docs](https://grafana.github.io/grafonnet-lib/api-docs/) for more information, but there are quick an easy ways to add new panels.
+
+### Here is an example of a panel, which has a few parameters that determine what it looks like in the dashboard.
+
+```
+local panelName = graphPanel.new(
```

Comment:
I would probably link out to line numbers in existing dashboards instead, which would make it easier to keep this documentation up to date